src/featureFlags.js

```diff
@@ -99,4 +110,28 @@ export class FeatureFlags extends events.EventEmitter {
   static async clearIgnoreWalletServiceFlag() {
     await localStorage.removeItem(IGNORE_WALLET_SERVICE_FLAG);
   }
+
+  /**
+   * Uses the Hathor Unleash Server and Proxy to determine if the
+   * wallet should have the Atomic Swap feature
+   *
+   * @return {boolean} The result from the unleash feature flag
+   */
+  async shouldUseAtomicSwap() {
+    try {
+      await this.client.updateContext({ userId: this.userId });
```

Comment:
This is the old way of handling feature flags, now we use properties as well. Check the wallet service method.

Reply:
Updated the context code on [82c5818](https://github.com/HathorNetwork/hathor-wallet/pull/361/commits/82c58188caa9f61f3b76f49098cd805545fb00ee).